packages/fluent-theme/src/fluent/styles/Pivot.styles.ts

```diff
@@ -0,0 +1,47 @@
+import { IPivotStyleProps } from 'office-ui-fabric-react/lib/Pivot';
+import { AnimationVariables } from 'office-ui-fabric-react/lib/Styling';
+
+export const PivotStyles = (props: IPivotStyleProps) => {
+  const { theme, rootIsTabs } = props;
+  const { palette } = theme;
+
+  return {
+    link: [
+      !rootIsTabs && {
+        height: 44,
+        lineHeight: 44,
+        selectors: {
+          ':hover': {
+            backgroundColor: palette.neutralLighter,
+            color: palette.neutralDark
+          },
+          ':active': {
+            backgroundColor: palette.neutralLight
+          }
+        }
+      }
+    ],
+    linkIsSelected: [
+      !rootIsTabs && {
+        height: 44,
+        lineHeight: 44,
+        selectors: {
+          ':hover': {
+            backgroundColor: palette.neutralLighter
+          },
+          ':active': {
+            backgroundColor: palette.neutralLight
+          },
+          ':before': {
+            transition: `left ${AnimationVariables.durationValue2} ${AnimationVariables.easeFunction2},
```

Comment:
There are Fluent variables for motion, but we don't have any guidance from the motion team at this time. So it seems reasonable to stick with the old animations for now.

Reply:
Animation info is here: https://microsoft.sharepoint.com/:x:/r/teams/SharedDesign/_layouts/15/Doc.aspx?sourcedoc=%7Be2e4a437-314a-4d81-94e8-a5e118d6a9cf%7D&action=default&uid=%7BE2E4A437-314A-4D81-94E8-A5E118D6A9CF%7D&ListItemId=91266&ListId=%7BD3B6CFCE-6C51-425D-B673-89233C78D523%7D&odsp=1&env=prodbubble

Reply:
**Selection indicator (underline)**
Scale | scale | decelerate | 400 ms | tabs | slow: tabs

**Page below if we can do it:**
  | Slide | position change | decelerate | 400 ms |   |  

Reply:
@betrue-final-final Please leave it to the PR's author to merge, so that any final changes can be made first. These animations will need to be added with a separate PR.